fee_calculator/apps/calculator/fixtures/price.json

```diff
@@ -35468,7 +35468,7 @@
     "advocate_type": "JRALONE",
     "offence_class": null,
     "fee_type": 12,
-    "unit": "HALFDAY",
```

Comment:
just to note the regs do not specify fees for "Junior alone" so i guess we defer to CCR which treats it the same as Led Junior